File: BECore/app/model/ContentExperiment.js
/// <summary>
/// Model EXTJS ContentExperiment.
/// </summary>
Ext.define('QLDT.model.ContentExperiment', {
    extend: 'Ext.data.Model',
    idProperty: 'ID',
    idType: 'Guid',
    fields: [
        { name: "ID", type: "string" },
		{ name: "CompanyID", type: "string" },
		{ name: "AtCompany", type: "string" },
		{ name: "Description", type: "string" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: 'ToDate', type: 'date', defaultValue: new Date() },
        { name: 'ProjectID', type: 'string' },
        { name: 'CompanyName', type: 'string' },
        { name: 'IdFiles', type: 'string' }
    ]
});

File: MT.Model/Business/ContentExperiment.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region ContentExperiment:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a ContentExperiment:EntityBase.
    /// </summary>
    public class ContentExperiment : EntityBase
    {
        [Key]
        public Guid ID { get; set; }
        public Guid CompanyID { get; set; }
        public string AtCompany { get; set; }
        public string Description { get; set; }

        public DateTime ToDate { get; set; }

        public List<ProjectExperiment_AttachDetail> Details { get; set; }

        public Guid ProjectID { get; set; }

        public string CompanyName { get; set; }

        public string IdFiles { get; set; }
    }
    #endregion
}
